### PR TITLE
config_variables: Add new variable aliases

### PR DESCRIPTION
**Pull request by siavasht**

### CODE REVIEW

File: source/configuration_manual/config_file/config_variables.rst [hunk 1]
```diff
@@ -100,6 +100,14 @@ Mail service user variables:
 |          |                |                                                               |
 |          |                | .. versionadded:: v2.2.19                                     |
 +----------+----------------+---------------------------------------------------------------+
+|          | local_ip       | local IP address (alias of %l)                                |
+|          |                |                                                               |
```
Comment (sirainen) [inline source/configuration_manual/config_file/config_variables.rst:104]: https://github.com/dovecot/documentation/pull/165 wasn't merged, but v2.3.12 should be replaced with v2.3.13 in those places..

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/configuration_manual/config_file/config_variables.rst (hunk 1 above)
Comment (siavasht) [reply source/configuration_manual/config_file/config_variables.rst:108]: included a commit with correct version fix.

**Review by sirainen — APPROVED** [review]

File: source/configuration_manual/config_file/config_variables.rst [hunk 2]
```diff
@@ -134,6 +150,14 @@ Login variables:
 | %k       | ssl_security | TLS session security string. If HAProxy is configured and it    |
 |          |              | terminated the TLS connection, contains "(proxied)".            |
 +----------+--------------+-----------------------------------------------------------------+
+|          | lip          | Deprecated version of %{local_ip}                               |
+|          |              |                                                                 |
+|          |              | .. deprecated:: v2.3.14                                         |
++----------+----------------+---------------------------------------------------------------+
+|          | rip          | Deprecated version of %{remote_ip}                              |
+|          |              |                                                                 |
```
Comment (sirainen) [inline source/configuration_manual/config_file/config_variables.rst:158]: Wrong space / - counts in the tables.

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/configuration_manual/config_file/config_variables.rst (hunk 2 above)
Comment (siavasht) [reply source/configuration_manual/config_file/config_variables.rst:158]: Fixed. Also notices some other formatting issues. Those should be fixed now as well.

**Review by sirainen — APPROVED** [review]

**Pull request merged**